website/docs/r/container_node_pool.html.markdown

```diff
@@ -89,11 +89,34 @@ resource "google_container_cluster" "primary" {
 }
 
 ```
+
+### Usage with a regional cluster
+
+```hcl
+
+resource "google_container_cluster" "regional" {
+  name = "marcellus-wallace"
+  region = "us-central1"
+}
+
+resource "google_container_node_pool" "regional-np" {
+  name       = "my-node-pool"
+  region       = "us-central1"
```

Comment:
nit: `=` alignment

Reply:
I actually appreciate all your attention to detail. This TF code is very vast! 👍